<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceWidget</class>
 <widget class="QWidget" name="ReferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>32</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="uiModel"/>
   </item>
   <item>
    <widget class="QLineEdit" name="uiText"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QToolButton" name="pushOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushNew">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushClear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="common.qrc"/>
 </resources>
 <connections/>
</ui>
